src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md

```diff
@@ -0,0 +1,80 @@
+## Spawn Patterns
+In version 3.3.0 we completely changed how aircraft coming into the system are defined.  We introduced **_Spawn Patterns_**.  Spawn Patterns provide a simple, consistent way to describe aircraft coming into the system.  Spawn Patterns are used for _both_ arrivals and departures.  The shape of the data is exactly the same for both, all keys are expected to be passed all the time.  
+
+Lets look at some examples before we continue:
+```javascript
+// Departures
+{
+    "airlines": [
+        ["aal", 10],
+        ["ual", 10],
+        ["ual/long", 3]
+    ],
+    "destination": "",
+    "origin": "KLAS",
+    "category": "departure",
+    "route": "KLAS.BOACH6.HEC",
+    "altitude": null,
+    "method": "random",
+    "rate": 5,
+    "speed": null
+}
+
+// Arrivals
+{
+    "airlines": [
+        ["aal", 10],
+        ["ual", 10],
+        ["ual/long", 3]
+    ],
+    "destination": "KLAS",
+    "origin": "",
+    "category": "arrival",
+    "route": "BETHL.GRNPA1.KLAS",
+    "altitude": [30000, 40000],
+    "method": "random",
+    "rate": 10,
+    "speed": 320
+}
+```
+## Key descriptions
+_(*) denotes a required field_
+
+#### airlines*
+List of airlines, and their spawn weight. A higher weight will increase the frequency that a particular airline is chosen. Airline selections are random based on weight.
+* Should always be a two-dimensional array
+* Should always have a shape of either:
+  - `[AIRLINE_ID, FREQUENCY_WEIGHT]`
+  - `[AIRLINE_ID/AIRLINE_FLEET, FREQUENCY_WEIGHT]`
+
+#### destination (* _for arrivals_)
+* destination airport `icao`
+
+#### origin (* _for departures_)
+* origin airport `icao`
+
+#### category*
+* Should be one of `arrival` or `departure`
+
+#### route*
+* Should be in the shape of a routeString.
+  - For SID/STAR procedures: `ENTRY.PROCEDURE_ID.EXIT`
```

Comment:
I'd really love to include a link or something here about what, exactly, a `routeString` is.  any good ones off the top of your head?